charts/spire/README.md

```diff
@@ -64,6 +64,14 @@ helm upgrade --install -n spire-mgmt spire-crds spire-crds --repo https://spiffe
 helm upgrade --install -n spire-mgmt spire spire --repo https://spiffe.github.io/helm-charts-hardened/ -f your-values.yaml
 ```
 
+### Clean up
```

Comment:
```suggestion
## Clean up
```

Reply:
Back to 2 so it is its own section at the same level as 'Install Instructions' and 'Upgrade notes' and then I think its good to go.